q2_fmt/_peds.py

```diff
@@ -365,33 +333,33 @@ def _compute_peds(peds_df: pd.Series, peds_type: str, peds_time: int,
                                 recip_df=recip_df,
                                 reference_column=reference_column)
     maskedrecip = donormask & recip_df
-    if peds_type == "Sample" or peds_type == "PPRS":
+    if type == "Sample" or type == "PPRS":
         num_sum = np.sum(maskedrecip, axis=1)
         donor_sum = np.sum(donormask, axis=1)
         for count, sample in enumerate(recip_df.index):
             sample_row = metadata.loc[sample]
-            peds_df.loc[len(peds_df)] = [sample, num_sum[count],
-                                         donor_sum[count],
-                                         sample_row[reference_column],
-                                         sample_row[subject_column],
-                                         sample_row[time_column]]
+            df.loc[len(df)] = [sample, num_sum[count],
+                               donor_sum[count],
+                               sample_row[reference_column],
+                               sample_row[subject_column],
+                               sample_row[time_column]]
 
-    elif peds_type == "Feature":
+    elif type == "Feature":
```

Comment:
Should this stay as-is? or type be changed to PRDF?

Reply:
I was keeping the internals the same because I figured it didn't matter, but I will change it